### DISCUSSION TITLE
How do I restore font used in Lawnchair 12?

### DISCUSSION THREAD

**Discussion by Shararamosh** [LawnchairLauncher/lawnchair#4825] [Q&A]
I've updated my Lawnchair to latest version using F-Droid and my font became different. I want to use exactly same font that was in Lawnchair 12, but I don't remember what font abd like size it was. How can I do it?

**Comment by validcube** [comment]
~~Lawnchair default font use the Inter font (3.00), but switching to the Inter font in the font switcher would likely use the latest version of the Inter font (4.00) which looks slightly different from default.~~

I'm dumb, you have to get into font switcher, select popup menus and reset to restore default font.

![Screenshot_20240916_114718_Lawnchair](https://github.com/user-attachments/assets/0811f6ad-e88e-460d-945f-b31e44573e05)

**Reply by Shararamosh** [reply]
The default font was still different to the one I previous had, as well as the distance between icons on home screen and apps list, so I ended up deleting version 14 and installing version 12.
